scan-triggers/aws-python-bucket-full-and-scheduled-scan/template.json

```
@@ -1,4 +1,29 @@
 {
+ "Metadata": {
+  "AWS::ServerlessRepo::Application": {
+   "Name": "cloudone-filestorage-plugin-trigger-full-scheduled-scan",
+   "Description": "It deploys all the required resources to trigger a full scan, scheduled or not, on a S3 bucket leveraging an existing Trend Micro File Storage Security deployment.",
+   "Author": "Trend Micro Cloud One File Storage Security",
```

With the upcoming rebranding do you want to go with:
```suggestion
   "Description": "It deploys all the required resources to trigger a full scan, scheduled or not, on a S3 bucket leveraging an existing Trend File Storage Security deployment.",
   "Author": "Trend Cloud One File Storage Security",
```